### PR TITLE
Make visits page the default page when you click a bar in the bar gra…

### PR DESCRIPTION
**Pull request by hardiesoft**
…ph frm the analysis page.

Added a bunny icon!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,7 +57,6 @@
     "babel-jest": "^25.1.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-syntax-async-functions": "^6.13.0",
-    "cacophony-types": "github:hardiesoft/cacophony-types#master",
```
Comment (clare) [inline package.json:59]: Should this be listed under hardiesoft?

File: src/views/AnalysisView.vue [hunk 2]
```diff
@@ -234,13 +235,13 @@ export default {
       }
       return str;
     },
-    gotoRecordingsSearchPage(array) {
-      const deviceName = array[0]._model.label;
+    gotoRecordingsSearchPage(chartItems: string[]) {
```
Comment (clare) [inline src/views/AnalysisView.vue:238]: Does we still need this function?

**Review by clare — APPROVED** [review]
Thanks Jon

File: package.json (hunk 1 above)
Comment (hardiesoft) [reply package.json:59]: I think this was something I was experimenting with a long time ago, and somehow got merged in somewhere along the way.  It's not being used, so I removed it.

**Pull request merged**